infrastructure_validations/aks/NHC/docker/aznhc_entrypoint.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash

set -euo pipefail
```

Enforces stricter error handling. The script fails eagerly.